libcloud/compute/types.py

```diff
@@ -247,6 +247,18 @@ class StorageVolumeState(object):
     UNKNOWN = 8
 
 
+class VolumeSnapshotState(object):
```

Comment:
In newer parts of the code we decided to replace numeric enum values with string values (e.g. RecordType enum).

Strings makes it easier to see what's going on since you don't need to check the enum type definition to see to which enum name a particular value maps to.

I would appreciate if you can update the code to use string values (e.g. AVAILABLE = 'available', ERROR = ' error', etc.).

In the future I also plan to clean and improve the legacy code and update `NodeState` to also use string values.